### DISCUSSION TITLE
[Icon Request] Game Controller

### DISCUSSION THREAD

**Discussion by ecklf** [tailwindlabs/heroicons#467] [Ideas]

**Comment by Im-Fran** [comment]
Also different kind of game controllers would be cool!
![image](https://github.com/tailwindlabs/heroicons/assets/30329003/fb19868f-c6af-4a88-bfe4-e746bcf99b04)
![image](https://github.com/tailwindlabs/heroicons/assets/30329003/aa1a3245-688d-4bfc-92ba-a81cad205377)
![image](https://github.com/tailwindlabs/heroicons/assets/30329003/9a4332f4-83ba-40df-90d6-ad6198419e58)

**Comment by ndri** [comment]
I added `gamepad` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=gamepad
